app/views/layouts/_header.html.erb

```diff
@@ -85,6 +85,12 @@
                   <% end %>
                 </li>
               <% end %>
+              <li>
+                <%= link_to "https://thunder-flower-8c2.notion.site/Casa-Volunteer-Tracking-App-HelpSite-3b95705e80c742ffa729ccce7beeabfa", target: :_blank do %>
```

Comment:
Here's a cool thing I learned recently. https://guides.rubyonrails.org/routing.html#direct-routes

What do you think about turning this into a direct route?

Reply:
Thanks @schoork for the feedback. Was not aware about it before, I have updated the code with direct route.